cli_tools/gce_onestep_image_import/main.go

```diff
@@ -47,37 +47,32 @@ func main() {
 func initLoggingParams(args *importer.OneStepImportArguments) service.InputParams {
 	return service.InputParams{
 		OnestepImageImportParams: &service.OnestepImageImportParams{
-			ImageImportParams: &service.ImageImportParams{
-				CommonParams: &service.CommonParams{
-					ClientID:                args.ClientID,
-					Network:                 args.Network,
-					Subnet:                  args.Subnet,
-					Zone:                    args.Zone,
-					Timeout:                 args.Timeout.String(),
-					Project:                 *args.ProjectPtr,
-					ObfuscatedProject:       service.Hash(*args.ProjectPtr),
-					Labels:                  fmt.Sprintf("%v", args.Labels),
-					ScratchBucketGcsPath:    args.ScratchBucketGcsPath,
-					Oauth:                   args.Oauth,
-					ComputeEndpointOverride: args.ComputeEndpoint,
-					DisableGcsLogging:       args.GcsLogsDisabled,
-					DisableCloudLogging:     args.CloudLogsDisabled,
-					DisableStdoutLogging:    args.StdoutLogsDisabled,
-				},
-				ImageName:          args.ImageName,
-				DataDisk:           args.DataDisk,
-				OS:                 args.OS,
-				NoGuestEnvironment: args.NoGuestEnvironment,
-				Family:             args.Family,
-				Description:        args.Description,
-				NoExternalIP:       args.NoExternalIP,
-				SourceFile:         args.SourceFile,
-				StorageLocation:    args.StorageLocation,
+			CommonParams: &service.CommonParams{
```

Comment:
may I know why you are not embedding the whole ImageImport args anymore? Is it becuase there are too many useless fields?

Reply:
yes, that's why